resources/views/components/game/top-achievers/score-row.blade.php

```diff
@@ -12,8 +13,16 @@
 
         <td class="text-right">{{ $rank }}</td>
         <td class="whitespace-nowrap">
-            <x-user.avatar display="icon" icon-size="xs" :user="$user" />
-            <x-user.avatar hasHref="true" :user="$user" />
+            {!! userAvatar($user, iconSize:16) !!}
+            <!-- <x-user.avatar display="icon" icon-size="xs" :user="$user" /> -->
+            <!-- <x-user.avatar hasHref="true" :user="$user" /> -->
```

Comment:
If we want to leave these in, recommend replacing them with `{{-- --}}` style comments.

`<!-- -->` gets shipped to the browser (marginally increasing the document size), whereas the Blade rendering engine strips `{{-- --}}`.